scope.go

```diff
@@ -235,6 +238,28 @@ func (s *scope) inc() error {
 		return err
 	}
 
+	// packet size token limit
+	if s.user.packetSizeTokenLimitBurst > 0 {
+		tl := s.user.packetSizeTokenLimiter
+		ctx, cancel := context.WithTimeout(context.TODO(), s.user.maxExecutionTime)
+		defer cancel()
+		err = tl.WaitN(ctx, int(formatPacketSizeByUnit(s.requestBodyBytes, s.user.packetSizeUnit))) // request packet size num by packet size unit
+		if err != nil {
+			return fmt.Errorf("limits for user %q is exceeded: packet_size_token_limit_burst limit: %d%s",
```

Comment:
We should make sure we handle query Counter decrementing here and below

Reply:
thanks for your suggestion，i would fix it